### PR TITLE
Loosen timing restrictions on regions caching test.

### PR DESCRIPTION
**Pull request by michaelcfanning**
@ScottLouvau.  This is failing non-deterministically in some agent environments. I've loosened the timing window and we now emit the actual timing observed in log file.

@lgolding @rtaket @nickmarston @cfaucon 

Could use a review on this. The test that's modified is causing random pipeline failures, so good to resolve asap.

### CODE REVIEW

File: src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs [hunk 1]
```diff
@@ -478,9 +478,11 @@ public void FileRegionsCache_ProperlyCaches()
                 Region populated = fileRegionsCache.PopulateTextRegionProperties(copy, uri, populateSnippet: true);
             }
 
+            int timeThreshold = 150;
```
Comment (lgolding) [inline src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs]: `const`?

Comment (michaelcfanning) [reply src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs]: :v:

Comment (ScottLouvau) [reply src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs]: Got it. Sorry about that. I need to find a new model for performance tests which are probably relative to measured performance of something else just run in the same environment.

Comment (michaelcfanning) [reply src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs]: Side note: in the agent 'wild', this code path is taking closer to 200 ms. You have a fine idea, though, we could update the test to run a baseline operation (no caching), then repeat with caching. We should see a substantive difference that's roughly consistent in relative terms. Will look at that.

---
In reply to: [397266682](https://github.com/microsoft/sarif-sdk/pull/1822#discussion_r397266682) [](ancestors = 397266682)

**Pull request merged**